### ISSUE TITLE
Virtual environment size & coordinates

### ISSUE THREAD

**Issue by jafphd** [deepmind/lab#188]
I am trying to understand some of the features of the virtual dimensions of a defined arena. In particular, when defining the map in the DeepMind Lab, what is the virtual length of an '*' (representing a wall segment) ?
Also, where I can get information about the coordinate system using during the navigation in that environment? (e.g. where is the origin of that coordinate system?)
Any help on these issues would be appreciated. Thanks!

EXAMPLE: 
self._map = make_map.makeMap{
        mapName = 'simple_arena',
        mapEntityLayer =
            '***************\n' ..
            '*                     *\n' ..
            '*        *           *\n' ..
            '*        *           *\n' ..
            '*        *           *\n' ..
            '*        *           *\n' ..
            '*        *           *\n' ..
            '*  G    *   P      *\n' ..
            '*        *            *\n' ..
            '*        *            *\n' ..
            '***************\n',
    }

**Comment by charlesbeattie** [comment]
Yes the `cellSize` defaults to 100. This can be changed by adding a cellSize=<integer> to `makeMap`. This is not well documented or tested as a lot of code was built with the assumption that this will be 100. The default is declared here:
https://github.com/deepmind/lab/blob/7b851dcbf6171fa184bf8a25bf2c87fe6d3f5380/deepmind/level_generation/text_level/text_level_settings.h#L185

And read from in makeMap kwargs here:
https://github.com/deepmind/lab/blob/29d157a4a14b44c470749f548379ae1b7e4b4ab6/deepmind/engine/lua_text_level_maker.cc#L362

I'm sorry that it is not documented in markdown. This is probably because of inadequate testing of the feature.

**Comment by jafphd** [comment]
@charlesbeattie Thank you very much for the information! It seems then that defining cellSize I would end up with walls with the same length. But I saw some videos of the DeepMind Lab environments having different length/thickness of walls. If you know how to do it, could you please indicate how/where to define the different wall properties?   
BTW, I cannot see a way to get the exact virtual coordinates where the wall were placed (I can only infer that information from the make_map.makeMap definition, right?
All the best!
